### ISSUE TITLE
After executing migrate:run, no table was created?

### ISSUE THREAD

**Issue by qingfengmy** [typeorm/typeorm#9851]
### Issue description

After executing migrate:run, no table was created?

### Expected Behavior

the tables was created.

### Actual Behavior

no tables was created.

### Steps to reproduce

When I'm ready to deploy the production environment. i run below script
```
npx typeorm-ts-node-commonjs migration:generate 
	./src/migration/init 
	-d ./src/data-source.ts 
	-t
```
and the init.ts is generated.
```
import { MigrationInterface, QueryRunner } from "typeorm";

export class init1678700370492 implements MigrationInterface {
    name = 'init1678700370492'

    public async up(queryRunner: QueryRunner): Promise<void> {
        await queryRunner.query(`CREATE TABLE \`user1\` (\`id\` int NOT NULL AUTO_INCREMENT, \`firstName\` varchar(255) NOT NULL, \`lastName\` varchar(255) NOT NULL, \`age\` int NOT NULL, PRIMARY KEY (\`id\`)) ENGINE=InnoDB`);
    }

    public async down(queryRunner: QueryRunner): Promise<void> {
        await queryRunner.query(`DROP TABLE \`user1\``);
    }

}

```

and i delete all tables in my local mysql.

and i run below script
```
npx typeorm-ts-node-commonjs migration:run -d ./src/data-source.ts
```
in my mysql, only migration_custom_table wa created. the user table was not created.

How should I fix this？

### My Environment

| Dependency          | Version  |
| ---                 | ---      |
| Operating System    |          |
| Node.js version     | 16  |
| Typescript version  | 4  |
| TypeORM version     | 0.3.12  |

### Relevant Database Driver(s)

- [ ] aurora-mysql
- [ ] aurora-postgres
- [ ] better-sqlite3
- [ ] cockroachdb
- [ ] cordova
- [ ] expo
- [ ] mongodb
- [X] mysql
- [ ] nativescript
- [ ] oracle
- [ ] postgres
- [ ] react-native
- [ ] sap
- [ ] spanner
- [ ] sqlite
- [ ] sqlite-abstract
- [ ] sqljs
- [ ] sqlserver

### Are you willing to resolve this issue by submitting a Pull Request?

Yes, I have the time, and I know how to start.

**Comment by Talalaev** [comment]
yep, I have the same bug on postgreSql database driver. And I think it is not dependent from DB. So I fixed it for my project just downgrade to 0.3.10 version.
